src/js/components/Clock/stories/typescript/CustomAnalog.tsx

```diff
@@ -38,8 +37,4 @@ const CustomAnalog = () => (
   </Grommet>
 );
 
-if (!isChromatic()) {
-  storiesOf('TypeScript/Clock', module).add('Custom Analog', () => (
-    <CustomAnalog />
-  ));
-}
+storiesOf('Clock', module).add('Custom Analog', () => <CustomAnalog />);
```

Comment:
This should actually be disabled on Chormatic, since the time is changing every second and we'll always have chromatic changes to approve if this story is not disabled.

Reply:
Disabled CustomAnalog and Digital stories on chromatic in the latest update